website/docs/r/codedeploy_deployment_group.html.markdown

```diff
@@ -327,7 +327,7 @@ The `target_group` configuration block supports the following:
 
 The `test_traffic_route` configuration block supports the following:
 
-* `listener_arns` - (Required) List of Amazon Resource Names (ARNs) of the load balancer listeners.
+* `listener_arns` - (Required) List of Amazon Resource Names (ARNs) of the load balancer listeners. The list has a maximum size of one.
```

Comment:
Perhaps "There must be exactly one item in this list" would be more clear, as this suggests an empty list is acceptable.